app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -241,7 +241,10 @@ public void onBackPressed() {
             contributionsFragment.backButtonClicked();
         } else if (nearbyParentFragment != null && activeFragment == ActiveFragment.NEARBY) {
             // Means that nearby fragment is visible
-            nearbyParentFragment.backButtonClicked();
+            if(nearbyParentFragment.backButtonClicked()==false) { //this function return false mean bottomsheet is not expand then if backbutton pressed then come to Contributions tab
```

Comment:
Could you please:
1) Write `! nearbyParentFragment.backButtonClicked()` instead of `nearbyParentFragment.backButtonClicked()==false` (it is a usual convention)
2) Move the comment to the line before (so that no line is too long)
Thanks!